docbook/reference/en/en-US/modules/Json.xml

```diff
@@ -1,63 +1,59 @@
 <chapter id="json">
     <title>JSON Support via Jackson</title>
-    <para>Besides the Jettision JAXB adapter for JSON, Resteasy also support integration with the Jackson project.
-        Many users find the output from Jackson much much nicer than the Badger format or Mapped format provided by
-        Jettison.
-        Jackson lives at http://jackson.codehaus.org. It allows you to easily marshal Java objects to and from JSON.
-        It has a Java Bean based model as well as JAXB like APIs. Resteasy integrates with the JavaBean model as
-        described
-        at this url: http://jackson.codehaus.org/Tutorial.
+    <para>Besides the Jettision JAXB adapter for JSON, Resteasy also supports integration with the Jackson project.
+        Many users find the output from Jackson much nicer than the Badger format or Mapped format provided by Jettison.
+        For more on Jackson 2, see <ulink url="http://wiki.fasterxml.com/JacksonHome">http://wiki.fasterxml.com/JacksonHome</ulink>.
+        Besides JAXB like APIs, it has a JavaBean based model, described at
+        <ulink url="http://wiki.fasterxml.com/JacksonDataBinding">http://wiki.fasterxml.com/JacksonDataBinding</ulink>,
+        which allows you to easily marshal Java objects to and from JSON.
+        Resteasy integrates with the JavaBean model.
+        While Jackson does come with its own JAX-RS integration, Resteasy expanded it a little, as decribed below.
     </para>
-    <para>While Jackson does come with its own JAX-RS integration, Resteasy expanded it a little. To include
-        it within your project, just add this maven dependency to your build.  Resteasy supports both Jackson 1.9.x
-        and Jackson 2.2.x.  Read further on how to use each.
+    <para>
+    <emphasis role="bold">NOTE.</emphasis> The resteasy-jackson-provider module, which is based on the outdated Jackson 1.9.x, is
+    currently deprecated, and will be removed in a release subsequent to 3.1.0.Final. The resteasy-jackson2-provider
+    module is based on Jackson 2. 
     </para>
+    
     <sect1>
-        <title>Using Jackson 1.9.x Outside of JBoss AS7</title>
-        <para>If you're deploying Resteasy outside of JBoss AS7 add the resteasy jackson provder to your WAR
-        pom.xml build</para>
+        <title>Using Jackson 1.9.x Outside of Wildfly 10 or EAP 7</title>
+        <para>If you're deploying Resteasy outside of Wildfly 10 or EAP 7,

```

Comment:
Minor note: I would stay general here and say "WildFly or JBoss EAP", without versions.